docs/reference/commandline/build.md

```diff
@@ -96,22 +96,22 @@ For example, run this command to use a directory called `docker` in the branch
 `container`:
 
 ```bash
-$ docker build https://github.com/docker/rootfs.git#container:docker
+$ docker build https://github.com/docker/rootfs#container:docker
```

Comment:
I don't think this change is correct; the error message indicates that `git` is not installed on your machine, which is needed to checkout the linked repository;

```
unable to find 'git': exec: "git": executable file not found in %PATH%
```

For example, this just works:

```bash
$ docker build -t repro-433 https://github.com/thaJeztah/pgadmin4-docker.git#1.4

Sending build context to Docker daemon  169.5kB
Step 1/8 : FROM python:2-alpine
 ---> c80455665c57
Step 2/8 : ENV PGADMIN_VERSION 1.4 PYTHONDONTWRITEBYTECODE 1
 ---> Running in cf5c0caf0ef4
 ---> affe9f50eeb0
Removing intermediate container cf5c0caf0ef4
....
```